packages/stream-tile/src/tile-document.ts

```diff
@@ -344,8 +344,13 @@ export class TileDocument<T = Record<string, any>> extends Stream {
       newContent = this.content
     }
 
-    if (header.controllers && header.controllers?.length !== 1) {
-      throw new Error('Exactly one controller must be specified')
+    if (header.controllers) {
+      if (header.controllers?.length !== 1) {
```

Comment:
```suggestion
      if (header.controllers.length !== 1) {
```

We (and TS compiler) already know, that `header.controllers` is defined.